### ISSUE TITLE
iMX8M mimi EVK occasionally goes into emergency shell

### ISSUE THREAD

**Issue by diegorusso** [ARMmbed/meta-mbl#684]
### Description

Sometimes when a board is flashed/booted it goes into Emergency Shell instead of getting to the normal login prompt.

```
[[0;32m OK [0m] Started [0;1;39mEmergency Shell[0m. 
[[0;32m OK [0m] Reached target [0;1;39mEmergency Mode[0m. 
[[0;32m OK [0m[ 23.108982] Atheros 8031 ethernet 30be0000.ethernet-1:00: attached PHY driver [Atheros 8031 ethernet] (mii_bus:phy_addr=30be0000.ethernet-1:00, irq=POLL) 
] Reached target [0;1;39mSockets[ 23.124382] IPv6: ADDRCONF(NETDEV_UP): eth0: link is not ready 
[0m. 
[ 23.132760] A link change request failed with some changes committed already. Interface eth0 may have been left with an inconsistent configuration, please check. 
[[0;32m OK [0m] Started [0;1;39mNetwork Service[0m. 
[[0;32m OK [0m] Started [0;1;39mCreate Volatile Files and Directories[0m. 
[[0;32m OK [0m] Started [0;1;39mRebuild Dynamic Linker Cache[0m. 
[[0;32m OK [0m] Started [0;1;39mRebuild Journal Catalog[0m. 
Starting [0;1;39mUpdate is Completed[0m... 
Starting [0;1;39mNetwork Time Synchronization[0m... 
Starting [0;1;39mRun pending postinsts[0m... 
Starting [0;1;39mNetwork Name Resolution[0m... 
[[0;32m OK [0m] Started [0;1;39mUpdate is Completed[0m. 
[[0;32m OK [0m] Started [0;1;39mRun pending postinsts[0m. 
[[0;32m OK [0m] Started [0;1;39mNetwork Time Synchronization[0m. 
[[0;32m OK [0m] Reached target [0;1;39mSystem Time Synchronized[0m. 
[[0;32m OK [0m] Started [0;1;39mNetwork Name Resolution[0m. 
[[0;32m OK [0m] Reached target [0;1;39mNetwork[0m. 
[[0;32m OK [0m] Reached target [0;1;39mHost and Network Name Lookups[0m. 
[ 27.358964] fec 30be0000.ethernet eth0: Link is Up - 1Gbps/Full - flow control off 
[ 27.366569] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready 
You are in emergency mode. After logging in, type \"journalctl -xb\" to view 
system logs, \"systemctl reboot\" to reboot, \"systemctl default\" or \"exit\" 
to boot into default mode. 
Press Enter for maintenance 
(or press Control-D to continue): [ 79.934303] Host SW:4.5.23.1, FW:0.0.0.27, HW:QCA93x7_REV1_1
```

Looking at the boot log, the following might be part of the problem:

```
Mounting [0;1;39m/var/log[0m... 
[ 22.001364] EXT4-fs (mmcblk1p9): ext4_check_descriptors: Checksum for group 0 failed (34404!=39447) 
[ 22.010451] EXT4-fs (mmcblk1p9): group descriptors corrupted! 
Mounting [0;1;39m/boot[0m... 
Mounting [0;1;39m/mnt/flags[0m... 
Mounting [0;1;39m/scratch[0m... 
Mounting [0;1;39m/home[0m...
```

### Issue request type

    [ ] Question
    [ ] Enhancement
    [x] Bug

**Comment by ciarmcom** [comment]
*Internal Jira reference: https://jira.arm.com/browse/MBLTRIAGE-24*

**Comment by jjohnson-arm** [comment]
Fixed in mbl-os-0.9
